cucumber/api/features/authenticate.feature

```diff
@@ -20,6 +20,11 @@ Feature: Exchange a role's API key for a signed authentication token
       cucumber:user:alice successfully authenticated with authenticator authn
     """
 
+  Scenario: Authenticating with no Content-Type header succeeds without leakage.
+    And I save my place in the log file
+    And I can authenticate Alice with no Content-Type header
+    Then The Alice's API key does not appear in the log
```

Comment:
There seems to be an extra `The` in this line...is this working? I would have thought that would cause it to not match the step def.

Reply:
😆 will fix.

It works because my regex isn't anchoring to start `^` and and end `$`:
```
And(/Alice's API key does not appear in the log/)
```